Source/Blazorise/Infrastructure/EventCallbackSubscriber.cs

```diff
@@ -3,63 +3,59 @@
-    /// and automatically unsubscribes from earlier <see cref="EventCallbackSubscribable{T}"/> instances
-    /// whenever it moves to a new one.
-    /// </summary>
-    public class EventCallbackSubscriber<T> : IDisposable
-    {
-        #region Members
+namespace Blazorise.Infrastructure;
 
-        private readonly EventCallback<T> handler;
+/// <summary>
+/// Represents an event callback that may be subscribed to an <see cref="EventCallbackSubscribable"/>.
+/// </summary>
+public class EventCallbackSubscriber : IDisposable
+{
+    #region Members
 
-        private EventCallbackSubscribable<T> existingSubscription;
+    private readonly EventCallback handler;
 
-        #endregion
+    private EventCallbackSubscribable existingSubscription;
 
-        #region Constructors
+    #endregion
 
-        /// <summary>
-        /// A default <see cref="EventCallbackSubscriber{T}"/> constructor.
-        /// </summary>
-        /// <param name="handler"></param>
-        public EventCallbackSubscriber( EventCallback<T> handler )
-        {
-            this.handler = handler;
-        }
+    #region Constructors
 
-        #endregion
+    /// <summary>
+    /// A default <see cref="EventCallbackSubscriber"/> constructor.
+    /// </summary>
+    /// <param name="handler"></param>
+    public EventCallbackSubscriber( EventCallback handler )
+    {
+        this.handler = handler;
+    }
 
-        #region Methods
+    #endregion
 
-        /// <summary>
-        /// Creates a subscription on the <paramref name="subscribable"/>, or moves any existing subscription to it
-        /// by first unsubscribing from the previous <see cref="EventCallbackSubscribable{T}"/>.
-        ///
-        /// If the supplied <paramref name="subscribable"/> is null, no new subscription will be created, but any
-        /// existing one will still be unsubscribed.
-        /// </summary>
-        /// <param name="subscribable"></param>
-        public void SubscribeOrMove( EventCallbackSubscribable<T> subscribable )
-        {
-            if ( subscribable != existingSubscription )
-            {
-                existingSubscription?.Unsubscribe( this );
-                subscribable?.Subscribe( this, handler );
-                existingSubscription = subscribable;
-            }
-        }
+    #region Methods
 
-        /// <inheritdoc/>
-        public void Dispose()
+    /// <summary>
+    /// Creates a subscription on the <paramref name="subscribable"/>, or moves any existing subscription to it
+    /// by first unsubscribing from the previous <see cref="EventCallbackSubscribable"/>.
+    ///
+    /// If the supplied <paramref name="subscribable"/> is null, no new subscription will be created, but any
+    /// existing one will still be unsubscribed.
+    /// </summary>
+    /// <param name="subscribable"></param>
+    public void SubscribeOrMove( EventCallbackSubscribable subscribable )
```

Comment:
Maybe Resubscribe? Or, SubscribeOrReplace?

Reply:
I'm fine with either of those, I just didn't know what move meant.
If you use either of those then I understand that it might replace an existing entry.